### DISCUSSION TITLE
Auto update block type

### DISCUSSION THREAD

**Discussion by nivcoo** [BG-Software-LLC/SuperiorSkyblock2#805] [Approved Feature Requests] [answered]
Hello, i have planned to migrate my server from 1.12 to 1.18 (Without data saving) but all my menu use 1.12 syntax (data and type). Maybe can you create "migrator" when we use data to convert to new block (I think isn't hard but long enough in time).

**Comment by OmerBenGera** [comment, marked as answer]
Too much work for something you can do manually in a matter of minutes.
I am not gonna implement such thing, if someone else wants tho - I am not against it.
